src/ts/components/seekbar.ts

```diff
@@ -48,6 +50,60 @@ export interface SeekBarMarker {
   duration?: number;
 }
 
+enum SeekBarType {
+  Vod,
+  Live,
+  Volume
+}
+
+interface ArrowKeyControls {
+  left: () => void;
+  right: () => void;
+  up: () => void;
+  down: () => void;
+}
+
+const arrowKeyControls = (currentValue: number, set: (number: number) => any, range: {min: number, max: number}): ArrowKeyControls => {
```

Comment:
`set` is maybe not an ideal name 🤔 What about `applyFunction` / `valueUpdateFunction`